spec/models/user_can_create_account_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+    describe "DB table" do
+        it { is_expected.to have_db_column :name }
+        it { is_expected.to have_db_column :email }
+        it { is_expected.to have_db_column :encrypted_password }
+    end
+
+    describe "Validations" do
+        it { is_expected.to validate_presence_of :name }
+        it { is_expected.to validate_presence_of :email }
+        it { is_expected.to validate_presence_of :encrypted_password }
```

Comment:
you need o test for `  validates_uniqueness_of :name` here

Reply:
Done as well